indico/modules/search/client/js/components/results/Contribution.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {List, Icon} from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+import './Contribution.module.scss';
+
+const Contribution = ({title, url, startDt, eventURL, eventTitle, persons}) => (
+  <div styleName="contribution">
+    <List.Header>
+      <a href={url}>{title}</a>
+    </List.Header>
+    <List.Description styleName="description">
+      <List.Item styleName="high-priority">
+        {/* change to something that reminds an event */}
+        <Icon name="calendar check outline" />
+        <a href={eventURL}>{eventTitle}</a>
+      </List.Item>
+      <List.Item>
+        {persons.length !== 0 && (
+          <ul styleName="high-priority">
+            {persons.length > 1 ? <Icon name="users" /> : <Icon name="user" />}
+            {persons.map(item => (
+              <li key={item.id}>{item.title ? `${item.title} ${item.name}` : `${item.name}`}</li>
+            ))}
+          </ul>
+        )}
+      </List.Item>
+      <List.Item styleName="med-priority">
+        <Icon name="calendar alternate outline" />
+        {moment(startDt, 'YYYY-MM-DDZhh:mm').format('DD MMMM YYYY HH:mm')}
```

Comment:
you always want `HH` (24h format), never `hh` (12h format)

Reply:
and the timestamp format looks wrong. there shouldn't be a `Z` separating date and time, but either a space or a `T`

Reply:
Yes but with the current mock dataset these setup works.. maybe when the real data comes we can change it

Reply:
the `hh` is wrong in any case, and when I was downstairs with you I remember seeing `02:00` times for all results in one of categories using mock data. the format returned by the server is `2017-10-18 13:00:00+00:00` so i'm surprised that parsing it works at all...

Reply:
fixed it